scripts/issue/71/consolidate_badger.py

```diff
@@ -0,0 +1,145 @@
+        treasury_balance = badger_token.balanceOf(dao_treasury.address)
+        token = dao_treasury.token()
+        releaseTime = dao_treasury.releaseTime()
+        beneficiary = dao_treasury.beneficiary()
+        
+        # Upgrades logic and verifies storage
+        new_logic = registry.eth.logic.SimpleTimelockWithVoting
+        old_devProxyAdmin.upgrade(dao_treasury.address, new_logic)
+
+        assert treasury_balance == badger_token.balanceOf(dao_treasury.address)
+        assert token == dao_treasury.token()
+        assert releaseTime == dao_treasury.releaseTime()
+        assert beneficiary == dao_treasury.beneficiary()
+        # Beneficiary is currently the DAO_Agent
+        assert dao_treasury.beneficiary() == registry.eth.badger_wallets.fees
+
+        # Nonces are messed up on this multi - next in line is 78
+        old_multi.post_safe_tx(replace_nonce=78)
+
+    else:
+        # Simulates upgrade of contract
+        if simulation == "true":
+            new_logic = registry.eth.logic.SimpleTimelockWithVoting
+            old_devProxyAdmin.upgrade(dao_treasury.address, new_logic)
+
+        safe = GreatApeSafe(DEV_MULTISIG)
+        treasury_vault = GreatApeSafe(TREASURY_VAULT)
+        safe.take_snapshot(tokens=[registry.eth.treasury_tokens.BADGER])
+        treasury_vault.take_snapshot(tokens=[registry.eth.treasury_tokens.BADGER])
+
+        # Contracts needed
+        controller = interface.IController(
+            registry.eth.controllers.native,
+            owner=safe.account
+        )
+        balance_checker = interface.IBalanceChecker(
+            registry.eth.helpers.balance_checker,
+            owner=safe.account
+        )
+        dao_treasury = interface.ISimpleTimelockWithVoting(
+            registry.eth.badger_wallets.DAO_treasury,
+            owner=safe.account
+        )
+        bBadger_vault = interface.ISettV4h(
+            registry.eth.sett_vaults.bBADGER,
+            owner=safe.account
+        )
+        bBadger_strat = interface.IStrategy(
+            registry.eth.strategies["native.badger"],
+            owner=safe.account
+        )
+
+        ## == Release all BADGER from DAO_Treasury to devMulti == ##
+        
+        # Sets new beneficiary
+        dao_treasury.setBeneficiary(DEV_MULTISIG)
+        assert dao_treasury.beneficiary() == DEV_MULTISIG
+
+        treasury_balance = badger_token.balanceOf(dao_treasury.address)
+        gov_balance = badger_token.balanceOf(DEV_MULTISIG)
+
+        # Release BADGER to devMultisig
+        dao_treasury.release()
+
+        balance_checker.verifyBalance(
+            badger_token.address,
+            DEV_MULTISIG,
+            treasury_balance + gov_balance
+        )
+
+        ## == Transfer missing BADGER to Geyser and withdrawAll to vault == ##
+        geyser_balance = badger_token.balanceOf(BADGER_GEYSER)
+        strategy_balanceOf = bBadger_strat.balanceOf()
+        geyser_deficit = strategy_balanceOf - geyser_balance
```

Comment:
i would add some extra room here, in case of activity on the strat. like `* 1.1` or something?

Reply:
Confirmed: Running the script yesterday, estimated a deficit of: `201661.80644761855 BADGER`. Today, after a few withdrawals already, the deficit is still the same: `201661.80644761855 BADGER`.

Reply:
@sajanrajdev this is to turn off earns for bbadger?

Reply:
@btcookies, yes, we are deprecating the strat so there's no point on keep earning it